confd/production.tmpl

```diff
@@ -162,17 +162,8 @@
         ]
     },
     "contextConfig": {
-		"elastic": {
-            "uri": "{{ getv "/ansyn/context/db/elastic/url" "NO-VALUE" }}",
-			"bucket": "context",
-			"auth": "elastic:changeme"
-		},
-		"proxy": {
-			"uri": "{{ getv "/ansyn/context/db/proxy/url" "NO-VALUE" }}",
-			"bucket": "contexts",
-			"apiObject" : "Http"
-		}
-    },
+    	"baseUrl": "{{ getv "/ansyn/layers/baseurl" "NO-VALUE" }}"
```

Comment:
Maybe we should merge all those env variables into one? We have a key for cases and layers.